position_control:
  - robot 0:
     k_linear: 0.5
     k_angular: 1
  - robot 1:
     k_linear: 0.5
     k_angular: 1
  - robot 2:
     k_linear: 0.5
     k_angular: 1

